Extension/lib/filter/rules/css-filter-rule.js

```diff
@@ -137,20 +145,30 @@ var CssFilterRule = exports.CssFilterRule = (function () {
             this.loadDomains(domains);
         }
 
+        var isExtendedCss = false;
         var cssContent = rule.substring(indexOfMask + mask.length);
 
         if (!isInjectRule) {
             var pseudoClass = parsePseudoClass(cssContent);

```

Comment:
We'd better add a comment with task link here. I have already almost forgot why we do it:)